fs2-aws-s3/src/main/scala/fs2/aws/s3.scala

```diff
@@ -176,6 +188,9 @@ object s3 {
                   .through(uploadPart(uploadId))
                   .fold[List[(PartETag, PartId)]](List.empty)(_ :+ _)
                   .through(completeUpload(uploadId))
+                  .handleErrorWith(ex =>
```

Comment:
I'm not sure that suppressing all exceptions is good idea, maybe we need to `rethrow` the exception after abortion?